draft-ietf-wimse-arch.md

```diff
@@ -77,17 +77,42 @@ Attestation is the function through which a task verifies the identity of a sepa
 
 # Architecture
 
-## Workload Identity
+## Workload Identity {#whimsical-identity}
 
-Workload identity often comprises multiple attributes that describe various aspects of a workload. These attributes can encompass diverse sets of information, including the workload's role within a system, the software it operates, and the hardware environment it utilizes. Different authorities across various parts of the system define these attribute sets. This architecture introduces a Workload Identifier for representing and referencing workloads, enabling diverse authorities with distinct identity attribute sets to consistently reference the workload.
+The identity of a workload is a complex topic.  The WIMSE architecture document starts by defining a trust domain, a workload identifier and workload identity credentials. These constructs are sufficient for establishing basic authentication, authorization and accounting. More complex identity constructs can be created from these basic building blocks.
```

Comment:
I would suggest to re-word it as
Workload identity construct consists of three basic building blocks: trust domain, workload identifier and workflow identity credentials. These components are sufficient for establishing authentication, authorization and accounting processes.

Reply:
Updated